apps/labs/posts/improving-accessibility-in-jupyterlab.md

```diff
@@ -0,0 +1,85 @@
+ 
+## 90 days, 1 goal: Making accessibility a reality
+My task of improving accessibility in JupyterLab required me to bind the two. I started off by reading articles and listening to talks. Learning about accessibility widened my thoughts it’s not the people, rather the environment that needs to be fixed.
+ 
+Accessibility is a big road to go down. And accessibility issues can come from a wide diversity of people and abilities. Some of them could be permanent like hearing impairment, low vision, locomotory issues while others could be temporary or situational. To make web content accessible, there are a set of guidelines called [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) and these are standard rules to keep in mind while designing your website. Integrating these will help make content on the web accessible, because of course the web is for all! :)
+ 
+ 
+We couldn’t cover all types of accessibility issues, so we just focused one corner that is [1.4.10](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html). This section works specifically for people who have varying abilities in terms of vision. They might need to enlarge their screens or use magnification softwares.
+ 
+Accessibility guidelines recommend two main features for this group.
+Your app or website should allow:
+- zoom up till 400%
+- scroll only in the vertical direction
+ 
+## Challenges
+On the journey of making JupyterLab more accessible, we needed to make sure JupyterLab moves parallel to WCAG, the standard guidelines. For this, JupyterLab should support zooming up till 400% and scrolling only in the vertical direction. But, when zooming in JupyterLab to that extent, we ran into issues. Some parts started breaking while others were hard to reach. We tried to identify areas for improvements and worked towards fixing them.
+ 
+## The Issue
+The gif below shows one of the issues we encountered. On browser zoom, some sections of the Running Panel disappear if sections above them are expanded. This makes `Language Servers` and `Terminals` impossible to reach at browser zoom. Mainly because there is no way to go down the panel.
+ 
+![Alt - The gif shows some sections of the Running Panel disappear if sections above them are expanded. This makes Language Servers and Terminals hard to reach at browser zoom](/posts/improving-accessibility-in-jupyterlab/before.gif)
+ 
+ 
+## The Fix
+ 
+We tried changing things here and there, and lastly putting a vertical scrollbar to the Running Panel worked!  I was happy, as I am most of the times. :)
+ 
+This gif shows a fix for the issue. After putting a scroll to the Running Panel, all sections are approachable even when other sections are expanded. Now, it’s easy to reach the bottom.
+ 
+ 
+![Alt - This gif shows a fix for the issue. We can see a vertical scroll bar to the Panel. All the sections of the Running Panel are now accessible and it’s easy to reach the bottom](/posts/improving-accessibility-in-jupyterlab/before.gif)
```

Comment:
```suggestion
![](/posts/improving-accessibility-in-jupyterlab/after.gif)
```

No need for the alt text here because the image description is already in the document.

I also fixed the path here: `after.gif` not `before.gif` ;)